src/ui/dialogs.py

```
@@ -16,28 +16,59 @@
 )
 from src.ui.custom_widgets import CustomNavButton
 
+# Suppress SSL certificate warnings when verify=False is used in requests
+try:
+    import urllib3
+except ModuleNotFoundError:
+    # Fallback: urllib3 may be vendored inside requests in some environments / PyInstaller builds
+    import requests.packages.urllib3 as urllib3  # type: ignore
```

Pylint (3.3.7)

[refactor] Use 'from requests.packages import urllib3' instead

(R0402)